mlflow/utils/autologging_utils.py

```diff
@@ -510,17 +517,20 @@ def __call__(self, original, *args, **kwargs):
                 raise e
 
 
+AutologgingSession = namedtuple("AutologgingSession", ["integration", "id"])
```

Comment:
Pull this out as a "public" developer API. Folks who supply their own `AutologgingEventLogger` implementations may have to reason about its content